package.json

```diff
@@ -57,8 +57,9 @@
         "lint-staged": "^12.5.0",
         "glob": "^10.3.3",
         "moment": "^2.29.4",
-        "monaco-editor": "0.39.0",
-        "monaco-yaml": "4.0.4",
+        "monaco-editor": "0.44.0",
+        "monaco-yaml": "5.1.1",
+        "react-monaco-editor": "^0.55.0",
```

Comment:
Should we add this to peer deps as well to ensure only single copy is created?

Reply:
Fixed for now. although we will eventually remove these from dashboard